src/scss/constants.scss

```diff
@@ -271,3 +275,25 @@ $z-layers: (
     text-decoration: underline $hover-color;
   }
 }
+
+@mixin over-image-style {
```

Comment:
Please suggest another way to cope with CSS code duplication that is commonly used in the codebase. One member of the team suggested mixins to me and that is why I used it here. To prevent code duplication.

Reply:
1. `global.scss`
2. Extract the element into a component. The fact that there are 2 elements in different places and they need the same styles, implies that the element should be a component with it's own styles/scss.

Reply:
Extracted `ImageOverlay` component in https://github.com/sparkletown/sparkle/pull/1988/commits/9e002122383637f650ba5292a75d3f9cea4b05c5